compiler/ir/backend.js/src/org/jetbrains/kotlin/ir/backend/js/JsLoweringPhases.kt

```diff
@@ -675,7 +691,8 @@ private val primitiveCompanionLoweringPhase = makeBodyLoweringPhase(
 private val callsLoweringPhase = makeBodyLoweringPhase(
     ::CallsLowering,
     name = "CallsLowering",
-    description = "Handle intrinsics"
+    description = "Handle intrinsics",
+    prerequisite = setOf(stringPlusLongLowering)
```

Comment:
This one lowers the `String::plus` to `jsPlus`, so the fix for Longs needs to happen first